apps/api/src/user/service/user.service.ts

```diff
@@ -30,7 +30,18 @@ export class UserService {
   }
 
   async getSelf(user: User) {
-    return user
+    const defaultWorkspace = await this.prisma.workspace.findFirst({
+      where: {
+        ownerId: user.id,
+        isDefault: true
+      },
+      select: {
```

Comment:
Hey, what I meant to say in here is, we don't need any custom selection. We want the entire workspace object to be returned.

Reply:
Fixed it